master/buildbot/test/fake/latent.py

```diff
@@ -42,15 +42,18 @@ class LatentController(SeverWorkerConnectionMixin):
     stop_instance() is executed.
     """
 
+    STATE_NOT_STARTED = 0
```

Comment:
would it make sense to use automat to implement this?

https://pypi.org/project/Automat/

Reply:
I think that using automat would make code less readable. It could probably work for `LatentController`, but in `AbstractLatentWorker` the '.state' member does not represent entire state and we make decisions about the next state taking into account more data than that. I'm probably biased, but in this case procedural paradigm allows easier building of the mental model than a declarative one.

By the way, I saw somewhere within Buildbot state constants being represented in a separate class as `NamedConstant` members. Should I do that here too?

Reply:
Since it's 3.5+, what about using enum?

Reply:
OK, will use enum.

Reply:
Should be fixed now.